.github/ISSUE_TEMPLATE/document-breakage.md

```diff
@@ -7,10 +7,12 @@ assignees: ''
 
 ---
 
-- <link to page with breakage> (last verified YYYY-MM-DD)
-  - <description of breakage>
+<!-- Each exception shoudl be documented. Please create an issue using template below before creating a new exception. -->
+<!-- Title should contain eTLD+1 of the tracker (and not the site that's broken). -->
+  
+- https://example.com <!--link to page with breakage--> (last verified 2021-12-31)
+  - Page is blank for 5s after load. <!-- description of breakage -->
   - requests required:
-    - `<request>` (`<type(s)>`) - <additional description>
-    - ...
-  - ...
-- ...
+    - `https://tracker.com` <!-- url of the request that needs to be unblocked --> (`js` <!--resource type(s)-->) - <!--additional description-->
+    - …
+- … <!-- other site that's broken due to blocking of the same tracker -->
```

Comment:
comments on a different line would be good, otherwise we have a lot of trailing space